system/jupyterhub/templates/cronjob-availability.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v1
```

Comment:
For completeness the CronJob should also be wrapped in the `{{- if .Values.jupyterhub_cron.enabled }}`